### PR TITLE
Feature/thank you page changes

### PR DESCRIPTION
**Pull request by ziek-za**
As per https://praekelt.atlassian.net/browse/ONE-41

### CODE REVIEW

File: org/templates/basic/org/share_site_tile.html [hunk 1]
```diff
@@ -16,7 +11,7 @@
             </a>
         </div>
         <div class="twitter">
-            <a href="http://www.twitter.com/share?url={{ request.get_host|iriencode }}"
+            <a href="http://www.twitter.com/share?url={{ request.get_host|iriencode }}&text=Every%20day%20we%20wait%2C%20more%20people%20die.%20Sign%20the%20petition%20%26%20tell%20world%20leaders%3A%20%23EndEbola%20now!"
```
Comment (hedleyroos) [inline org/templates/basic/org/share_site_tile.html]: The message won't render nicely. You're missing a %20. Did you test it?

Comment (ziek-za) [inline org/templates/basic/org/share_site_tile.html]: I have tested it, it comes out as expected. Is it fine to hardcore it like that?

Comment (hedleyroos) [inline org/templates/basic/org/share_site_tile.html]: EndEbola is not a word.

Comment (ziek-za) [inline org/templates/basic/org/share_site_tile.html]: Sorry for the misunderstanding, it is a hashtag (#EndEbola)

**Comment by hedleyroos** [comment]
+1

**Pull request closed without merging**